src/Tempest/Log/src/FileHandlers/RotatingFileHandler.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Log\FileHandlers;
+
+use DateTimeImmutable;
+use InvalidArgumentException;
+use Monolog\Handler\RotatingFileHandler as MonoRotatingFileHandler;
+
+final class RotatingFileHandler extends MonoRotatingFileHandler
```

Comment:
I think this could be refactored to an enum, wouldn't that be cleaner?

Reply:
You want to convert `RotatingFileHandler::FILE_PER_WEEK` etc to enums... I can do this, are you sure we want to introduce that much changes to essentially internal monolog functionality, if it ever changes more will have to be updated in that case. Or am I misunderstanding something??

Reply:
Oh I totally missed the extend part 😂 No I get it now :)